src/utils/__tests__/reviewExpirationUtils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { determineExpirationTime } from '@utils/reviewExpirationUtils';
+
+describe('Review Expiration', () => {
+  const OLD_ENV = process.env;
+
+  afterAll(() => {
+    process.env = OLD_ENV;
+  });
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env.REQUEST_EXPIRATION_MIN = '60';
+    process.env.WORKDAY_START_HOUR = '8';
+    process.env.WORKDAY_END_HOUR = '17';
+  });
+
+  it('should pick a time starting on the next working day if would expire after 5 PM', () => {
+    // Monday, January 17th, 2022 at 4:30 PM
+    const date = new Date(2022, 0, 17, 16, 30, 0);
+
+    const expirationDate = new Date(determineExpirationTime(date));
+
+    // Tuesday, January 18th, 2022 at 9:00 AM
+    verifyDate(expirationDate, 0, 18, 9, 2022);
+  });
+
+  it('should pick a time starting on the next working day if would expire before 8 AM', () => {
+    // Monday, January 3rd, 2022 at 5:00 AM
+    const date = new Date(2022, 0, 3, 5, 0, 0);
+
+    const expirationDate = new Date(determineExpirationTime(date));
+
+    // Monday, January 3rd, 2022 at 9:00 AM
+    verifyDate(expirationDate, 0, 3, 9, 2022);
+  });
+
+  it('should pick a time on the next Monday if it would expire on the weekend', () => {
```

Comment:
@bable5 here's the test for weekends. I believe it should be working. Inside the implementation there's a `[1, 2, 3, 4, 5]` array to indicate skipping ahead days until we hit M, T, W, Th, F.